ls1intum/Artemis
# [Bugfix] Resolve missing key in translation in Tutor Assessment

### Checklist

### Motivation and Context


The jhiAlert Service uses the translation Service to translate the received `key`. On some occasions, we want to directly use a specified message for which no translation key is available. 

### Description

This fixes a false parameter which was passed into the jhiAlert error function.  
In case a translation key is not found, the previous output to the alert-component was : 

translation-not-found[translation.key]

The approach now is to remove the `translation-not-found[]` portion and just show the translation key. 
We sent additionally a warning to Sentry to be informed if we are missing such a key.

### Steps for Testing


1. Log in to Artemis
2. Try to generate a warning message
3. For example create a Programming exercise, with a title and shortname which already exists
4. Verify that the warning message is working as intended and no issue was sent to Sentry. 
(Only issues with a missing translation key shall send a notification to Sentry)

### Test Coverage

No new relevant code or features were added, so the test coverage stays the same.

### Screenshots



Correct alert notification:
![image](https://user-images.githubusercontent.com/33484318/103890102-c5bb8380-50e7-11eb-8586-13993ebbb804.png)

New Issue in Sentry:
![image](https://user-images.githubusercontent.com/33484318/103889155-3d88ae80-50e6-11eb-8e81-5896a1d9e431.png)
